packages/browser/src/browser/index.ts

```diff
@@ -278,7 +284,16 @@ async function loadAnalytics(
 
   attachInspector(analytics)
 
-  const plugins = settings.plugins ?? []
+  const plugins = settings.plugins?.filter(
```

Comment:
Nit: Can we encapsulate this into the `registerPlugins` so:
- `loadAnalytics` has a habit of getting large (it does a lot of different things, it's kind of a god function), so I've been gradually trying to break it up.
- `registerPlugins` doesn't need a new argument, since it can be derived from `plugins`. It already has a huge signature that's weird (opts + options?? LOL)

Reply:
Done